app/src/main/res/layout/fragment_news_details.xml

```diff
@@ -0,0 +1,148 @@
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/vNewsDetailsSwipeRefreshLayout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/wolox_white"
+    android:orientation="vertical">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+
+        <FrameLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <ImageView
+                android:id="@+id/vNewsDetailsBackgroundNew"
+                android:layout_width="match_parent"
+                android:layout_height="@dimen/new_details_image_height" />
+
+            <androidx.appcompat.widget.Toolbar
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:background="@drawable/top_news_details_gradient">
+
+                <ImageView
+                    android:id="@+id/vNewsDetailsBackButton"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="@dimen/spacing_large"
+                    android:layout_marginTop="@dimen/spacing_large"
+                    android:layout_marginEnd="@dimen/spacing_large"
+                    android:layout_marginBottom="@dimen/spacing_large"
+                    android:src="@drawable/ic_back" />
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="9dp"
+                    android:layout_marginTop="@dimen/spacing_large"
+                    android:layout_marginEnd="@dimen/spacing_large"
+                    android:layout_marginBottom="@dimen/spacing_large"
+                    android:maxLines="1"
+                    android:text="@string/vNewsDetailsTopText"
+                    android:textAlignment="textStart"
+                    android:textAllCaps="false"
+                    android:textSize="@dimen/font_medium_extra"
+                    android:textStyle="bold"
+                    android:verticalScrollbarPosition="defaultPosition" />
+
+            </androidx.appcompat.widget.Toolbar>
+
+        </FrameLayout>
+
+        <ScrollView
+            android:layout_width="match_parent"
+            android:layout_height="match_parent">
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/spacing_tiny"
+                android:layout_marginEnd="@dimen/spacing_tiny">
+
+                <ImageView
```

Comment:
What is this for?

Reply:
The design example shows margin
![image](https://user-images.githubusercontent.com/19233953/65550415-e7462b00-def5-11e9-958a-36caf37eb8b1.png)